### PR TITLE
makes science less secure by removing one airlock door

### PR DESCRIPTION
**Pull request by athena413**
# General Documentation

### Intent of your Pull Request

makes the main entrance to science less secure by removing an airlock door, since the main entrance to sci is pretty secure :(

also compresses the research area (i put the toolboxes over in the testing lab)

image
after

image
before

### Why is this change good for the game?

it's now less annoying to leave science, and now science's main entrance is now no longer as secure/almost as secure as the sec main entrance

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
it's a mapping change to science

### What should players be aware of when it comes to the changes your PR is implementing?
the map is different for science

### What general grouping does this PR fall under? 
mapping

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
no

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
n/a

# Changelog

:cl:  
tweak: science map changes
/:cl:

### CODE REVIEW

**Comment by PrinceKirze** [comment]
sci has the most potential to fuck the round so it's the most secure
no i dont think this change will screw over millions of rounds to come but i see no reason to make it less secure when sci has 
1) plasma cans (engi dont even got these)
2) toxins (plasma tanks, so singletanks)
3) nanites (fuck a round up rq if they're already installed in people)
4) idk they can lathe a lot of shit
5) also all of the RD's shit i guess

**Comment by UselessTheremin** [comment]
this is super ugly too

**Comment by athena413** [comment]
> 
> 
> sci has the most potential to fuck the round so it's the most secure
> no i dont think this change will screw over millions of rounds to come but i see no reason to make it less secure when sci has
> 
>     1. plasma cans (engi dont even got these)
> 
>     2. toxins (plasma tanks, so singletanks)
> 
>     3. nanites (fuck a round up rq if they're already installed in people)
> 
>     4. idk they can lathe a lot of shit
> 
>     5. also all of the RD's shit i guess

engi DOES have a plasma can but honestly yeah i guess thats fair
